sdk/core/src/services/sboms/service.rs

```diff
@@ -68,7 +68,8 @@ impl SbomService {
         provider: SbomProviderKind,
         xref: Xref,
         task: &Task,
-    ) -> Result<Sbom, Error> {
+    ) -> Result<Option<Sbom>, Error> {
```

Comment:
Why did we change this to an option?

Reply:
I changed it back. Ended up tracing it and it wasn't necessary